tests/ard_reprocessed_l1s/dev_go_reprocess.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# If a number is passed in it is assumed to be the scene limit
+# otherwise the default is 400
+
+set -o errexit
+set -o xtrace
+
+if [[ "$HOSTNAME" == *"gadi"* ]]; then
+	echo If module load breaks check out a clean environment
+	module use /g/data/v10/public/modules/modulefiles
+	module use /g/data/v10/private/modules/modulefiles
+  	
+	# remove for the production script
+	module use /g/data/u46/users/$USER/devmodules/modulefiles
+
+	# Update for the production script
+	module load ard-scene-select-py3-dea/dev_20230524
+	# module load ard-scene-select-py3-dea/20230522
+
+fi
+
+if [ -z "$1" ]
+  then
+    echo "No argument supplied"
+	scenelimitvalue=400
+else
+	scenelimitvalue=$1
+fi
+
+dry_run=" "
+runard="--run-ard"
```

Comment:
most of the variables for flags to ard_scene_select here are in snake case where `runard`, `dryrun` and `scenelimit ` are in camel case. 
 Should we consider on making it consistent?

Reply:
yes, thanks